docs/components/waitlist/overview.mdx

```diff
@@ -0,0 +1,349 @@
+---
+title: '`<Waitlist />` component'
+description: Clerk's <Waitlist /> component is used to render a waitlist form that allows users to join for early access to your application.
+---
+
+![Clerk's \<Waitlist /> component is used to render a waitlist form that allows users to join for early access to your application.](/docs/images/ui-components/waitlist.svg)
+
+Clerk's `<Waitlist />` component is used to render a waitlist form that allows users to join for early access to your application.
+
+> [!NOTE]
+> The `<Waitlist />` component is only available in Core 2. If you are using Next.js is available after <code>@clerk/nextjs\@6.2.0</code>
+
+## Properties
+
+All props are optional.
+
+<Properties>
+  - `appearance`
+  - <code>[Appearance](/docs/customization/overview) | undefined</code>
+
+  Optional object to style your components. Will only affect [Clerk components](/docs/components/overview) and not [Account Portal](/docs/customization/account-portal/overview) pages.
+
+  ---
+
+  - `afterJoinWaitlistUrl`
+  - `string`
+
+  Full URL or path to navigate after join waitlist.
+
+  ---
+
+  - `signInUrl`
+  - `string`
+
+  Full URL or path where the SignIn component is mounted.
```

Comment:
This is the description we use on the other component pages. Is it still accurate for the Waitlist component?

```suggestion
  Full URL or path to the sign in page. Use this property to provide the target of the 'Sign In' link that's rendered. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead.
```